releases/v1.6/schedule.md

```diff
@@ -0,0 +1,22 @@
+
+| **When**   | **Milestone**           | **Who**     | **What**                         | **Notes**                                                                                                              |
+|------------|-------------------------|-------------|----------------------------------|------------------------------------------------------------------------------------------------------------------------|
+| **March**  |                         |             |                                  |                                                                                                                        |
+| 2025/03/11 |                         | Kubernetes  | Tag 1.33.0-beta.0                |                                                                                                                        |
+| 2025/03/25 |                         | KubeVirt CI | Start the 1.33 provider          | Add on-demand pre-submit                                                                                               |
+| **April**  |                         |             |                                  |                                                                                                                        |
+| 2025/04/23 | Kubernetes Release      | Kubernetes  | 1.33.0 release                   |                                                                                                                        |
+| 2025/04/30 | KubeVirt Alpha 0 (1.6)  | KubeVirt    | Tag v1.6.0-alpha.0               |                                                                                                                        |
+| **May**    |                         |             |                                  |                                                                                                                        |
+| 2025/05/14 |                         | KubeVirt CI | CI lanes for provider are voting |                                                                                                                        |
+| 2025/05/21 | KubeVirt Beta 0         | KubeVirt    | Tag v1.6.0-beta.0                |                                                                                                                        |
+| 2025/05/21 | KubeVirt Beta 0         | KubeVirt CI | Mandatory pre-submit             | Non-standard lanes (i.e. migrations, multus, ipv6) are to be bumped by stakeholder                                     |
+| **June**   |                         |             |                                  |                                                                                                                        |
+| 2025/06/25 | KubeVirt Feature Freeze | KubeVirt    | Branch release-1.6               |                                                                                                                        |
```

Comment:
It just came to my mind that we talked about taking a look at the current list of quarantined tests and whether any of these would need to get back in before the release. IMHO we need to do that before the branch is cut, so at the latest at this point.
```suggestion
| 2025/06/25 | KubeVirt Feature Freeze | KubeVirt    | Branch release-1.6               | Review [list of quarantined tests](https://storage.googleapis.com/kubevirt-prow/reports/quarantined-tests/kubevirt/kubevirt/index.html)                                                                                                                         |
```

Reply:
Sure!
Done